components/details.tsx

```diff
@@ -22,15 +21,15 @@ export default function Details(props: DetailsProps) {
           label="Name" 
           value={info.name}
           required
-          onChange={(e: ChangeEvent) => props.updateName((e.target as HTMLInputElement).value)}
+          onChange={e => props.updateName(e.target.value)}
```

Comment:
Maybe add updateName and updateDesc to destructured props or remove them altogether